.env

```diff
@@ -10,3 +10,4 @@ HARVEST_SOURCE_URL=http://localhost:${HARVEST_SOURCE_PORT}
 
 CF_API_URL=https://api.fr.cloud.gov
 LM_RUNNER_APP_GUID=f4ab7f86-bee0-44fd-8806-1dca7f8e215a
+CKAN_API_URL=https://catalog-dev.data.gov
```

Comment:
why does local docker use catalog-dev url?

Reply:
good point. it shouldn't need that. it was for testing.

Reply:
updated